br/pkg/lightning/backend/local/local.go

```diff
@@ -1113,7 +1160,6 @@ func (local *Backend) generateAndSendJob(
 	jobWg *sync.WaitGroup,
 ) error {
 	logger := log.FromContext(ctx)
-	// TODO(lance6716): external engine should also support it
 	localEngine, ok := engine.(*Engine)
```

Comment:
Can we remove this type conversion?

Reply:
This is too get smaller region to improve concurrency when use dynamic region. I think external engine does not support to split again. If we want the region size smaller than the cluster configuration, we can use a smaller value when create RangeSplitter?

Reply:
Oh the problem is when use dynamic region, we should use large range size to split & scatter, but smaller range size to import. So we must split region again, I think we can postpone the fix to future PR.

Reply:
Maybe move this assignment to `if` statement to make it local to prevent it from being misused.